### ISSUE TITLE
Nightly update kills login cookies

### ISSUE THREAD

**Issue by languitar** [x86dev/docker-ttrss#30]
After every nightly update I have to log in again in the web interface. Is there a chance to work around this?

**Comment by x86dev** [comment]
Yes, by disabling or delaying the update mechanism ... have a look at update-ttrss.sh (at the end), gets invoked by service.d/ttrss-updater.

**Comment by languitar** [comment]
But what is it in the update process that kills the cookies? Are they invalidated with every new git commit?
